### ISSUE TITLE
WCAG AAA and Text-shadow

### ISSUE THREAD

**Issue by TharenaMelishka** [TheJaredWilcurt/UGUI-pa11y#60]
Question:

Does this program consider text-shadow to be a part of the font colour?  I've reworked my main colour to be above and beyond the 7.0 ratio, and it's still failing saying the same 3.46 or 1.11, or whatever it is ratio no matter how I change the colour.  nothing quite like running Koally and seeing 300+ errors over colour ratio.  

Thanks!

**Comment by TheJaredWilcurt** [comment]
should be based on the text color and the background color. it does not take into account shadow
